cli/src/installers/dependencyVersionMap.ts

```diff
@@ -16,6 +16,7 @@ export const dependencyVersionMap = {
   "drizzle-orm": "^0.28.5",
   "drizzle-kit": "^0.19.13",
   "dotenv-cli": "^7.3.0",
+  "mysql2": "^3.6.1",
```

Comment:
required for drizzle-kit as far as i can tell

Reply:
Yea i think i recall needing that in my devdeps 👍

Reply:
We are going to bundle all driver dependencies in drizzle-kit, so this one should become unnecessary

Reply:
I'll try to push this next, so you don't need to install anything